Cargo.toml

```diff
@@ -20,6 +20,7 @@ ic-agent = "0.15.0"
 ic-base-types = { git = "https://github.com/dfinity/ic", rev = "8ed0f58e65237ecf56982781a87c75053155ff66" }
 ic-nervous-system-common ={ git = "https://github.com/dfinity/ic", rev = "8ed0f58e65237ecf56982781a87c75053155ff66" }
 ic-sns-governance = { git = "https://github.com/dfinity/ic", rev = "8ed0f58e65237ecf56982781a87c75053155ff66" }
+ic-ic00-types = { git = "https://github.com/dfinity/ic", rev = "9ccfb7c3ed424e907be2213c3bb18fb0bd740efb" }
```

Comment:
`9ccfb7c3ed424e907be2213c3bb18fb0bd740efb` is behind `8ed0f58e65237ecf56982781a87c75053155ff66` ([9ccfb7](https://gitlab.com/dfinity-lab/public/ic/-/commit/9ccfb7c3ed424e907be2213c3bb18fb0bd740efb) vs [8ed0f5](https://gitlab.com/dfinity-lab/public/ic/-/commit/8ed0f58e65237ecf56982781a87c75053155ff66)). Can you bring this into parity or just grab the latest commit for everything?